crates/cdk/src/wallet/receive.rs

```diff
@@ -152,13 +154,21 @@ impl Wallet {
         mint_proofs.extend(p);
 
         let mut total_amount = Amount::ZERO;
-        for (mint, proofs) in received_proofs {
-            total_amount += Amount::try_sum(proofs.iter().map(|p| p.amount))?;
-            let proofs = proofs
+        for (mint, recv_proofs) in received_proofs {
```

Comment:
Unrelated to this actual PR so can do as a separate one., but i believe this hashmap and loop are unnecessary now that we only handle single mint tokens here. Opened an issue for it here https://github.com/cashubtc/cdk/issues/377